hash techniques for orientation/DCT-basedHashforRobustness.py

```diff
@@ -0,0 +1,51 @@
+# Import dependencies
+from PIL import Image
+import imagehash
+import numpy as np
+import glob
+import sys
+import os
+import csv
+
+#input and output file paths
+input_path1 = r'/Users/rakshithanagendrappa/Desktop/NormalOrientationMaps/*.jpg'
+out_loc = '/Users/rakshithanagendrappa/Desktop/OrientationRobustness/DCT-basedHashHammingDistances.txt'
+
+#Lists to store the values
+fingerPrintWithHD = []
+fingerPrintWithHDList = []  #list of lists
+
+image_paths1 = list(glob.glob(input_path1))
+for i in image_paths1:
+    l = i.split('/')
+    fileName = l[-1]
+    input_path2 = r'/Users/rakshithanagendrappa/Desktop/DistortedOrientationMaps/'
+    input_path2 = input_path2 + fileName
```

Comment:
using the approach above you can get rid of these things. See the link I gave.

Reply:
split and then concatenation is done to get the same image from the distorted folder as from normal folder